pallets/multisig/src/lib.rs

```diff
@@ -134,16 +182,14 @@ decl_storage! {
         /// A mapping of proposals to their IDs.
         pub ProposalIds get(fn proposal_ids):
             double_map hasher(twox_64_concat) T::AccountId, hasher(blake2_256) T::Proposal => Option<u64>;
-        /// Number of votes in favor of a tx. Mapping from (multisig, tx id) => no. of approvals.
-        pub TxApprovals get(fn tx_approvals): map hasher(twox_64_concat) (T::AccountId, u64) => u64;
         /// Individual multisig signer votes. (multi sig, signer, proposal) => vote.
         pub Votes get(fn votes): map hasher(blake2_128_concat) (T::AccountId, Signatory, u64) => bool;
         /// Maps a multisig to its creator's identity.
         pub MultiSigCreator get(fn ms_creator): map hasher(twox_64_concat) T::AccountId => IdentityId;
         /// Maps a key to a multisig address.
         pub KeyToMultiSig get(fn key_to_ms): map hasher(blake2_128_concat) AccountKey => T::AccountId;
-        /// Know whether the proposal is closed or not
-        pub ProposalClosed get(fn is_proposal_closed): map hasher(twox_64_concat) (T::AccountId, u64) => bool;
+        /// Details of a multisig proposal
+        pub ProposalDetail get(fn proposal_detail): map hasher(twox_64_concat) (T::AccountId, u64) => ProposalDetails<T::Moment>;
```

Comment:
Missing `OnRuntimeUpgrade` trait to support the migration.

Reply:
This PR targets v2 testnet which does not require migration. If we decide we want to backport this change, I'll add `OnRuntimeUpgrade` trait in a separate backporting PR that targets `aldebaran-staging`